packages/dns-test/src/container/network.rs

```
@@ -157,4 +194,30 @@ mod tests {
 
         Ok(())
     }
+
+    #[test]
+    fn stress() {
+        let mut networks = vec![];
```

This test fails on my Macbook. There are 5 other Docker networks present when I run this test.

---

are those networks created by default in a fresh macOS installation? what are their subnet masks? perhaps we should `docker network inspect` the output of `docker network ls` and use the subnet masks not shown there.

---

This is the collection of information on my Macbook.

```shell
$ docker network ls
NETWORK ID     NAME                             DRIVER    SCOPE
71ce00f0ea3c   bridge                           bridge    local
4ddc8f3c484c   customer-portal_cp_net           bridge    local
eb3ec8e5c769   download-server_criticalup_net   bridge    local
05a125eec965   host                             host      local
e40dd388a572   none                             null      local
```

Subnet masks for the first three entries are:

```
172.17.0.0/16
172.20.0.0/16
172.21.0.0/16
```